easycv/core/visualization/image.py

```diff
@@ -1,18 +1,119 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 # Adapt from https://github.com/open-mmlab/mmpose/blob/master/mmpose/core/visualization/image.py
 import math
+import os
+from urllib import request
 
 import cv2
 import mmcv
 import numpy as np
 from mmcv.utils.misc import deprecated_api_warning
+from PIL import Image, ImageDraw, ImageFont
+from torch.hub import DEFAULT_CACHE_DIR
+
+
+def download_font(save_path=None):
+    url_path = 'http://pai-vision-data-hz.oss-accelerate.aliyuncs.com/EasyCV/pkgs/simhei.ttf'
```

Comment:
add this font to git and auto package it into our python whl

Reply:
put simhei.ttf to resource dir